Resources/config/services.xml

```diff
@@ -43,10 +57,22 @@
             <argument type="service" id="event_dispatcher"/>
         </service>
         <!-- OpenSSL Key Loader -->
-        <service id="lexik_jwt_authentication.openssl_key_loader" class="%lexik_jwt_authentication.openssl_key_loader.class%">
-          <argument>%lexik_jwt_authentication.private_key_path%</argument>
-          <argument>%lexik_jwt_authentication.public_key_path%</argument>
-          <argument>%lexik_jwt_authentication.pass_phrase%</argument>
+        <service id="lexik_jwt_authentication.key_loader.openssl" class="%lexik_jwt_authentication.key_loader.openssl.class%">
+            <argument>%lexik_jwt_authentication.private_key_path%</argument>
+            <argument>%lexik_jwt_authentication.public_key_path%</argument>
+            <argument>%lexik_jwt_authentication.pass_phrase%</argument>
+        </service>
+        <!-- SecLib Key Loader -->
+        <service id="lexik_jwt_authentication.key_loader.seclib" class="%lexik_jwt_authentication.key_loader.seclib.class%">
+            <argument>%lexik_jwt_authentication.private_key_path%</argument>
+            <argument>%lexik_jwt_authentication.public_key_path%</argument>
+            <argument>%lexik_jwt_authentication.pass_phrase%</argument>
+        </service>
+        <!--- OpenSSL Key Loader / Old implementation (deprecated since 2.0) -->
+        <service id="lexik_jwt_authentication.openssl_key_loader" class="Lexik\Bundle\JWTAuthenticationBundle\Services\OpenSSLKeyLoader">
```

Comment:
@weaverryan I was glad to discover this feature, but as we have some tests running on old symfony versions, the tests don't pass anymore :/ So I had to rollback. I need to discuss with maintainers to know if we keep the compatibility with symfony versions older than 2.8. What do you think about this?